nestris_ocr/config.py

```diff
@@ -1,155 +1,4 @@
-from cached_property import threaded_cached_property
-from collections import OrderedDict
-import json
-
-from nestris_ocr.utils.sub_image import spawn_subimage
 from nestris_ocr.utils.program_args import args
+from nestris_ocr.config_class import Config
```

Comment:
`config_class` is a little too un-Python to me 😅 

How about turning `config.py` into a module instead? So:
```
nestris_ocr
  config.py          -> config/__init__.py
  config_class.py    -> config/config.py
```

Edit: Ah nvm, `config.py` would also clash with the `config` variable defined in this file. Let's refactor it later.

Reply:
Yeah naming was a pita 😔. Better suggestions welcome.

Reply:
I dont like the new names but i cant think of a better solution 👍